markdown/articles/2021/08/Non_blue_fair_decks.txt

```diff
@@ -0,0 +1,115 @@
+---
+authors: OtterLady
+banner: /assets/stinging-study.c21.jpg
+title: “Matchup Guides: Non Blue Fair Decks”
+---
+
+Gude!
+
+This time i want to talk how our Gameplan against Non Blue Fair Decks looks like and how to execute it.
+All these matchups have one common point, a fast Doomsday is always good and the post Doomsday part of the game is far more interesting.
+
+!EXCLAIMER! I don't give any sideboard guides in these articles. Sideboard guides don't help anyone learning to play a deck, because you first have to know how to play the deck to know how to sideboard correctly.
+
+##DnT
```

Comment:
Add a space after `#` in titles (and avoid abbreviations in titles)
```suggestion
## Death and Taxes
```

Reply:
changed